contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,518 @@
+---
+title: Identify
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+
+---
+
+Identify enables you to to track your users across different devices, platforms (frontend and backend), and connect events from before and after they log in for the first time.
```

Comment:
Identify doesn't really enable you to track users, `capture` tracks users. Identify is about adding or editing information to the users behind events.

You can track users without ever using `identify`

Reply:
I think the key phrase that's missing here is `the same user`